src/commands/toolchain.rs

```diff
@@ -42,6 +46,16 @@ pub struct UninstallCommand {
 #[derive(Debug, Parser)]
 pub struct ListRevisionsCommand {}
 
+#[derive(Debug, Parser)]
+pub struct ExportCommand {
+    /// Toolchain to export, [possible values: latest, beta-1, beta-2, beta-3, beta-4, nightly].
+    /// Your currently active toolchain will be exported if name isn't specified
```

Comment:
`Toolchain::from_settings()` gives the default toolchain, not the currently active one.

```suggestion
    /// The default toolchain will be exported if name isn't specified
```